src/Core/Billing/Queries/ISubscriberQueries.cs

```diff
@@ -6,6 +6,18 @@ namespace Bit.Core.Billing.Queries;
 
 public interface ISubscriberQueries
```

Comment:
Added two new methods to safely retrieve a Stripe `Customer` using a `ISubscriber`. This is used in the `CreateCustomerCommand`.